### ISSUE TITLE
LAcheck - Trace point sum != aligned interval

### ISSUE THREAD

**Issue by pb-cdunn** [PacificBiosciences/DALIGNER#21]
Reported by @rhallPB:

```
$ cd /dept/secondary/testdata/fail/lacheck
$ LAcheck -v raw_reads.db L1.117.137.las
  L1.117.137: Trace point sum != aligned interval
```

**Comment by pb-cdunn** [comment]
See discussion in thegenemyers/DALIGNER#43

**Comment by pb-cdunn** [comment]
Fixed in 8f179db7fb0bf59f34c4a073b85137a592336d89.
